tools/reindex-index-data

```diff
@@ -62,6 +81,7 @@ def reindex(index, doc_types):
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description='Regenerate index based on the existing data to update the index types.')
     parser.add_argument('index', choices=['transfers', 'aips'], help='Index to re-generate.')
+    parser.add_argument('--chunk-size', default=50, help='Number of docs in one chunk sent to Elasticsearch.')
```

Comment:
This should include `type=int` or `args.chunk_size` will be a string when not the default.